src/style.css

```diff
@@ -1,3 +1,67 @@
 body {
+  font-family: Roboto, serif, sans-serif;
```

Comment:
The `font-family` declaration should include a fallback font. Consider using a more specific fallback like `sans-serif` instead of listing both `serif` and `sans-serif`.